### ISSUE TITLE
round_num and block no longer allowed when querying balances

### ISSUE THREAD

**Issue by BunsanMuchi** [algorand/py-algorand-sdk#471]
### Your environment

`py-algorand-sdk    2.0.0`

### Steps to reproduce

1. Call the method `asset_balances(asset_id, round_num=number)` or `asset_balances(asset_id, block=number)`

### Expected behaviour

A balance return at the given block/round number according to the docs: https://py-algorand-sdk.readthedocs.io/en/latest/algosdk/v2client/indexer.html

### Actual behaviour

It seems like the method got changed, and doesn't allow for this parameter anymore, I believe it's a case of outdated docs or an undocumented change, since I get the following TypeErrors:

`TypeError: IndexerClient.indexer_request() got an unexpected keyword argument 'round_num'`
`TypeError: IndexerClient.indexer_request() got an unexpected keyword argument 'block'`

**Comment by tzaffi** [comment]
Thanks for identifying this issue. It appears that the docs getting generated are inaccurate and do not conform with the actual v2 API which [does not accept block/round number](https://developer.algorand.org/docs/rest-apis/indexer/#get-v2assetsasset-idbalances).

This discrepancy ought to be fixed as soon as possible, even though it probably is a longstanding issue.

**Comment by BunsanMuchi** [comment]
Yep, I've noticed it happens with a lot of places on both the SDK and the docs of the SDK. Plus the few methods that still support it end up with either rewind issues or just printing the current round, leading to confusing results. Thanks!

**Comment by tzaffi** [comment]
If you know off hand other instances, we welcome your feedback.

**Comment by tzaffi** [comment]
The fix didn't make it into today's release, but will go in the next one.
